scripts/run-once-scripts/postgres-migration/delete-user-outboxes.ts

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
This is basically just a copy of delete-section-outboxes with `user` replaced by `section`. I thought a) it would be better to separate the scripts for more granularity and b) it wasn't worth refactoring (e.g., have each call a "delete-outbox-script") since these are one-time scripts, and there are only two.